packages/kokoas-client/src/pages/projEstimate/estimateDataGrid/renderers/CustomAutomplete.tsx

```diff
@@ -0,0 +1,74 @@
+import { Autocomplete, TextField } from '@mui/material';
+import { RenderEditCellProps } from 'react-data-grid';
+import { RowItem } from '../useColumns';
+import { useEffect, useRef } from 'react';
+
+type CustomProps = {
+  data : string[] | undefined;
+};
+
+export const CustomAutomplete = ({
```

Comment:
```suggestion
export const CustomAutcomplete = ({
```
誤記でしょうか？
ファイル名も確認お願いします

Reply:
誤記です！ありがとうございます！